charts/deployment.yaml

```diff
@@ -0,0 +1,61 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: module-14
+spec:
+  replicas: 3
+  selector:
+    matchLabels:
+      app: module-14
+  template:
+    metadata:
+      labels:
+        app: module-14
+    spec:
+      containers:
+      - name: nginx
+        image: todo-app:prod
+        ports:
+        - containerPort: 80
```

Comment:
Note that this doesn't match your PORT environment variable below, so one of those will need tweaking for this to be accessible (and if this line, the appropriate targetPort in service.yaml too)